src/config.rs

```diff
@@ -59,6 +59,7 @@ const DEFAULT_PORT: &str = "5115";
 /// --no-error-summary         Doesn't display an error summary
 /// --no-print-metrics         Doesn't display metrics at end of load test
 /// --report-file NAME         Create an html-formatted report
+/// --no-granular-data         Disables granular data in HTML report graphs
```

Comment:
Would `--no-granular-report` make it more clear that this specifically impacts `--report-file` and no other data?